src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ComService.kt

```diff
@@ -42,16 +42,18 @@ class ComService(
    * Check if the username and staffId do not match. This should not happen, unless a Delius account is updated to point
    * at another linked account using the staffId. In this scenario, we should update the existing record to reflect
    * the new username and or staffId.
+   *
    */
+  @Deprecated("This can be removed after get in next caseload changes in, we can then rename this to a caseload service")
```

Comment:
Tried not too touch this much as there's going to be conflicts with the search ticket, as the comment says this can be deleted